src/commands/node.mjs

```diff
@@ -1467,9 +1470,13 @@ export class NodeCommand extends BaseCommand {
         .freezeWith(client)
         .execute(client)
 
-      this.logger.debug(`Upgrade frozen with transaction id: ${freezeUpgradeTx.transactionId.toString()}`)
+      const freezeUpgradeReceipt = await freezeUpgradeTx.getReceipt(client)
+
+      this.logger.debug(`Upgrade frozen with transaction id: ${freezeUpgradeTx.transactionId.toString()}`,
+        freezeUpgradeReceipt.status.toString())
     } catch (e) {
       this.logger.error(`Error in freeze upgrade: ${e.message}`, e)
+      await sleep(360000)
```

Comment:
what is the purpose for sleeping for 6 minutes?  when it throws the error, then it will exit the app and stop processing.

Reply:
was added during debug, removed it.